src/ServiceFabric/ServiceFabric/Models/ManagedClusters/NetworkSecurityDirectionEnum.cs

```diff
@@ -0,0 +1,22 @@
+﻿// ----------------------------------------------------------------------------------
+//
+// Copyright Microsoft Corporation
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+// http://www.apache.org/licenses/LICENSE-2.0
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+// ----------------------------------------------------------------------------------
+
+namespace Microsoft.Azure.Commands.ServiceFabric.Models
```

Comment:
Dont think its standard to add enum to the end of an enum type name, especially since you aren't doin this for all enums. If you are running into name collisions just add PS at the front like PSManagedCluster does. This advice goes for all type names

Reply:
enums updated